enhancements/machine-api/control-plane-set.md

```diff
@@ -0,0 +1,585 @@
+---
+title: control-plane-set-controller
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2022-01-11
+last-updated: 2022-01-17
+tracking-lin:
+  - TBD
+replaces:
+  - "[/enhancements/machine-api/control-plane-machinesets.md](https://github.com/openshift/enhancements/blob/master/enhancements/machine-api/control-plane-machinesets.md)"
+  - "https://github.com/openshift/enhancements/pull/278"
+  - "https://github.com/openshift/enhancements/pull/292"
+---
+
+# Control Plane Set Controller
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+To enable automated management of vertical scaling and replacement of Control Plane Machines, this proposal introduces
+a new resource and controller that will manage Control Plane Machine instances.
+
+## Motivation
+
+As OpenShift adoption increases and our managed services offerings become more popular, the manual effort required to
```

Comment:
may be mention/provide some context here on how a hypershift topology will solve this problem for multicluster scenarios and mention scenarios where this is still useful because a non hosted control plane is needed. e.g first management cluster.